pyproject.toml

```diff
@@ -43,6 +43,11 @@ Repository = "https://github.com/xdf-modules/pyxdf"
 Issues = "https://github.com/xdf-modules/pyxdf/issues"
 Changelog = "https://github.com/xdf-modules/pyxdf/blob/main/CHANGELOG.md"
 
+[project.optional-dependencies]
+playback = [
+    "pylsl>=1.17.6",
+]
+
```

Comment:
This should be removed.

Reply:
Is this not still useful for people who want to do `pip install "pyxdf[playback]"`?
(with the caveat that it's only seamless for Windows users or users who already have liblsl on their system)

Reply:
Sure, we can keep it!